tests/compute/test_specialization.py

```diff
@@ -418,18 +413,6 @@ def _afunc(nodes):
                  _afunc)
     assert F.allclose(o1, g.ndata.pop('o1'))
     assert F.allclose(o2, g.ndata.pop('o2'))
-    # multi builtins, one v2v spmv, one fallback to e2v, one fallback to degree-bucketing
-    g.update_all([fn.src_mul_edge(src='h', edge='w1', out='m1'),
-                  fn.src_mul_edge(src='h', edge='w2', out='m2'),
-                  fn.src_mul_edge(src='h', edge='w1', out='m3')],
-                 [fn.sum(msg='m1', out='o1'),
-                  fn.sum(msg='m2', out='o2'),
-                  fn.max(msg='m3', out='o3')],
-                 _afunc)
-    assert F.allclose(o1, g.ndata.pop('o1'))
-    assert F.allclose(o2, g.ndata.pop('o2'))
-    assert F.allclose(o3, g.ndata.pop('o3'))
```

Comment:
why is this test removed?

Reply:
Because there is no fallback to degree-bucketing anymore, the test is the same as testing multiple builtin functions (e.g. L404-L409)